zanshinsdk/client.py

```diff
@@ -199,92 +207,344 @@ def _request(self, method: str, path: str, params=None, body=None) -> httpx.Resp
-        <https://api.zanshin.tenchisecurity.com/#tag/Account>
+        <https://api.zanshin.tenchisecurity.com/#operation/getMe>
         :return: a dict representing the user
         """
         return self._request("GET", "/me").json()
 
+    def iter_invites(self) -> Iterator[Dict]:
+        """
+        Iterates over the invites of current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/getInvites>
+        :return: an iterator over the invites objects
+        """
+        yield from self._request("GET", "/me/invites").json()
+
+    def get_invite(self, invite_id: Union[UUID, str]) -> Dict:
+        """
+        Gets an specific invitation details, it only works if the invitation was made for the current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/getInviteById>
+        :param invite_id: the ID of the invite
+        :return: a dict representing the user invite
+        """
+        return self._request("GET", f"/me/invites/{validate_uuid(invite_id)}").json()
+
+    def accept_invite(self, invite_id: Union[UUID, str]) -> Dict:
+        """
+        Accepts an inivitation with the informed ID, it only works if the user accepting the invitation is the user that received the invitation.
+        <https://api.zanshin.tenchisecurity.com/#operation/acceptInviteById>
+        :param invite_id: the ID of the invite
+        :return: a dict representing the organization of this invite
+        """
+        return self._request("POST", f"/me/invites/{validate_uuid(invite_id)}/accept").json()
+
+    def iter_api_keys(self) -> Iterator[Dict]:
+        """
+        Iterates over the API keys of current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/getMyApiKeys>
+        :return: an iterator over the api keys objects
+        """
+        yield from self._request("GET", "/me/apikeys").json()
+
+    def create_api_key(self, name: Optional[str]) -> Dict:
+        """
+        Cretes a new API key for the current logged user, API Keys can be used to interact with the zanshin api directly on behalf of that user.
+        <https://api.zanshin.tenchisecurity.com/#operation/createApiKeys>
+        :param name: the Name of your new API key
+        :return: a dict representing the user api key
+        """
+        body = {
+            "name": name
+        }
+        return self._request("POST", "/me/apikeys", body=body).json()
+
+    def delete_api_key(self, api_key_id: Union[UUID, str]) -> bool:
+        """
+        Deletes a given API key by its id, it will only work if the informed ID belongs to the current logged user.
+        <https://api.zanshin.tenchisecurity.com/#operation/deleteApiKey>
+        :param api_key_id: the ID of the API key
+        :return: a boolean if success
+        """
+        return self._request("DELETE", f"/me/apikeys/{validate_uuid(api_key_id)}").json()
+
+    def get_freshdesk_id(self) -> Dict:
```

Comment:
Is this an operation our customers or integrations will need to use? Isn't this just for the SSO access to the Freshdesk web portal?

Reply:
yes, i will remove this function